.github/workflows/handle_release_tag.yaml

```diff
@@ -0,0 +1,34 @@
+name: Handle Release Tag
+
+on:
+  push:
+    tags:
+    - 'v*'
+
+jobs:
+  create_release:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+      with:
+        submodules: true
+        fetch-tags: true
+        fetch-depth: 0
+    - name: Repair tag
+      run: git fetch -f origin ${{ github.ref }}:${{ github.ref }}
+    - name: Verify that the tag is annotated
+      run: if test x$(git for-each-ref ${{ github.ref }} | awk '{print $2}') = xtag; then /bin/true; else echo "\"${{ github.ref }}\" does not look like an annotated tag!"; /bin/false; fi
+    - name: Submodule status
+      run: git submodule status
+    - name: Collect manifests
+      run: tools/collect-manifests.sh ~+/release-manifests
```

Comment:
what does the `~+` do?

Reply:
Gives the current directory, from root.  I tried used $PWD here but got an unexpected result when I did an 'echo $PWD'...just "***".  I dunno if it's really asterisks or if it's being hidden from me.   So I switched to ~+.